src/applications/edu-benefits/components/ReviewCardField.jsx

```diff
@@ -319,6 +319,9 @@ export default class ReviewCardField extends React.Component {
       this.props.formContext.onError();
     } else {
       this.setState({ editing: false, canCancel: true });
+      if (typeof get('onSaveClick', this.props.uiSchema) === 'function') {
+        this.props.uiSchema.onSaveClick();
```

Comment:
Does this need to be field on the form config? ReviewCardField is in the edu-benefits folder and only used on this form, it seems like you could call `recordEvent` directly in here.